src/vif.jl

```diff
@@ -0,0 +1,122 @@
+_rename_intercept(s) = strip(s) == "1" ? "(Intercept)" : s
+
+"""
+    termnames(model)
+
+Return the names associated with (fixed effects) terms in a model.
+
+For models with only continuous predictors, this is the same as
+`coefnames(model)`.
+For models with categorical predictors, the returned names reflect
+the categorical predictor and not the coefficients resulting from
+the choice of contrast coding.
+"""
+termnames(model) = collect(_rename_intercept.(string.(formula(model).rhs.terms)))
+function termnames(model::MixedModel)
+    return collect(_rename_intercept.(string.(formula(model).rhs[1].terms)))
+end
+
+_terms(model) = collect(formula(model).rhs.terms)
+_terms(model::MixedModel) = collect(formula(model).rhs[1].terms)
```

Comment:
IIUC this can be

```suggestion
termnames(model) = collect(_rename_intercept.(string.(_terms(model))))

_terms(model) = formula(model).rhs.terms
_terms(model::MixedModel) = formula(model).rhs[1].terms
```

Both `collect`s shouldn't be necessary but I think one of them likely is if the `.terms` is a tuple or something.

Reply:
Here, it converts the tuple to a vector for consistency with `coefnames`.